### PR TITLE
Add cli command contao:crawl

### PR DESCRIPTION
**Pull request by fkaminski**
Some translations are not so elegant. Suggestions are welcome ...

### CODE REVIEW

File: docs/manual/cli/crawl.de.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+weight: 20
+---
+
+
+{{< version "4.9" >}}
+
+Contao ist mit einem HTTP-Crawler ausgestattet. Intern basiert dieser auf [Escargot](https://github.com/terminal42/escargot).
+Der Crawler durchforstet im wesentlichen alle von Contao generierten URLs, wie jede andere Suchmaschiene. Er folgt Links, 
+die Teil der `sitemap.xml` sind, respektiert `robots.txt` Informationen, das `rel`-Attribut bei Links und vieles mehr.
+Dabei können eine beliebige Anzahl von sogenannten »Subscribern« die Ergebnisse der HTTP Anfragen abonnieren und
```
Comment (netzarbeiter) [inline docs/manual/cli/crawl.de.md]:
```suggestion
Dabei können eine beliebige Anzahl von sogenannten »Subscribern« die Ergebnisse der HTTP-Anfragen abonnieren und 
```

File: docs/manual/cli/crawl.de.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+aliases:
+    - /de/cli/crawl/
+weight: 20
+---
+
+
+{{< version "4.9" >}}
+
+Contao ist mit einem HTTP-Crawler ausgestattet. Intern basiert dieser auf [Escargot](https://github.com/terminal42/escargot).
+Der Crawler durchforstet im wesentlichen alle von Contao generierten URLs, wie jede andere Suchmaschiene. Er folgt Links,
```
Comment (netzarbeiter) [inline docs/manual/cli/crawl.de.md]:
```suggestion
Der Crawler durchforstet im wesentlichen alle von Contao generierten URLs, wie jede andere Suchmaschine. Er folgt Links, 
```

File: docs/manual/cli/crawl.de.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+die Teil der `sitemap.xml` sind, respektiert `robots.txt` Informationen, das `rel`-Attribut bei Links und vieles mehr.
+Dabei können eine beliebige Anzahl von sogenannten »Subscribern« die Ergebnisse der HTTP Anfragen abonnieren und 
+grundsätzlich beliebig weiterverarbeiten. Zum jetzigen Zeitpunkt kennt Contao zwei Abonnenten:
+
+* `search-index` - Aktualisiert den integrierten Suchindex (nur verfügbar, wenn die Suche überhaupt aktiviert wurde)
+* `broken-link-checker` - Überprüft alle Seiten auf fehlerhafte Links
+
+Jede Erweiterung kann weitere »Subscriber« bereitstellen, so dass diese Liste nicht unbedingt abschließend ist.
+
+```sh
```
Comment (netzarbeiter) [inline docs/manual/cli/crawl.de.md]:
```suggestion
```bash
```

File: docs/manual/cli/crawl.de.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+$> php vendor/bin/contao-console contao:crawl [options] [<job>]
```
Comment (netzarbeiter) [inline docs/manual/cli/crawl.de.md]:
```suggestion
php vendor/bin/contao-console contao:crawl [options] [<job>]
```

File: docs/manual/cli/crawl.en.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+requests and basically process them further in any way they want. As of today, Contao knows
+two subscribers:
+
+* `search-index` - updates the built-in search index (only available if searching was even enabled)
+* `broken-link-checker` - checks all the pages for broken links
+
+Any extension might provide additional subscribers so this list is not necessarily conclusive.
+Use it as follows:
+
+```sh
```
Comment (netzarbeiter) [inline docs/manual/cli/crawl.en.md]:
```suggestion
```bash
```

File: docs/manual/cli/crawl.en.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+$> php vendor/bin/contao-console contao:crawl [options] [<job>]
```
Comment (netzarbeiter) [inline docs/manual/cli/crawl.en.md]:
```suggestion
php vendor/bin/contao-console contao:crawl [options] [<job>]
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Review by netzarbeiter — APPROVED** [review]

**Review by fritzmg — APPROVED** [review]

**Pull request merged**